class/defaults.yml

```diff
@@ -3,7 +3,7 @@ parameters:
     =_metadata:
       multi_instance: true
     namespace: syn-nfs-subdir-external-provisioner
-    release_name: ${_instance}
+    name: ${_instance}
```

Comment:
I wouldn't change the component parameter since we're still configuring the Helm release name.

If you do replace the parameter without continuing to support the old parameter, you have to treat this PR as a breaking change since you might break configurations which overwrite the parameter, cf. https://syn.tools/syn/explanations/commodore-components/major.html#_replacing_a_component_parameter